src/pykeen/datasets/base.py

```diff
@@ -713,6 +718,90 @@ def _load_validation(self) -> None:
         pass  # already loaded by _load()
 
 
+class ZipFileSingleDataset(LazyDataset):
```

Comment:
The [SingleTabbedDataset](https://pykeen.readthedocs.io/en/stable/api/pykeen.datasets.base.SingleTabbedDataset.html#pykeen.datasets.base.SingleTabbedDataset) should be able to read a zipped TSV file. Does this not work because of the weird directory structure for BioKG?

Reply:
It did indeed work - I was just getting ahead of myself and didn't look at SingleTabbedDataset properly. It might be cool to document somewhere that it can deal with zip files etc.